docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,146 @@
+
+Using the $OIDC_ID of the issuer found above, you can check if a provider is installed with the following command:
+
+```shell
+aws iam list-open-id-connect-providers | grep $OIDC_ID
+```
+
+If there is no output, you will need to create an OIDC provider:
+
+```shell
+eksctl utils associate-iam-oidc-provider --cluster $CLUSTER_NAME --region $REGION --approve
+```
+
+### IAM Role
+
+To provide the necessary permissions, an IAM role and an attached policy are required. The role name is passed as an argument during the *helm* deployment.
+
+Create the role with a *role name* of your choosing (for example, `rancher-csp-iam-role`), and the required policy attached to it:
+
+```shell
+eksctl create iamserviceaccount \
+  --name rancher-csp-billing-adapter \
+  --namespace cattle-csp-billing-adapter-system \
+  --cluster $CLUSTER_NAME \
+  --region $REGION \
+  --role-name $ROLE_NAME --role-only \
+  --attach-policy-arn 'arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess' \
+  --approve
+```
+
+## Installing Rancher  
+
+1. Log helm into the AWS Marketplace Elastic Container Registry (ECR) to fetch the application. The AWS Marketplace ECR is always in the `us-east-1` region:
+
+  ```shell
+  export HELM_EXPERIMENTAL_OCI=1
+
+  aws --region us-east-1 ecr get-login-password \
+    | helm registry login --username AWS \
+    --password-stdin 709825985650.dkr.ecr.us-east-1.amazonaws.com
+  ```
+
+1. Install Rancher into your cluster using `helm`. Customize your `helm` installation values if needed:
+
+  :::note
+
+  Rancher Prime utilizes cert-manager to issue and maintain its certificates. Rancher will generate a CA certificate of its own, and sign a cert using that CA.
+
+  :::
+
+  The Rancher hostname must be resolvable by a public DNS. Please refer to the [Prerequisites](./rancher-prime-aws.md#prerequisites) section for more details. For example, if the DNS name is rancher.my.org, HOST_NAME=rancher.my.org.
+
+  ```shell
+  helm install -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \
+  oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/$REPOSITORY/rancher-cloud-helm/rancher-cloud \
+    --version $CHART_VERSION \
+    --set rancherHostname=$HOST_NAME \
+    --set rancherServerURL=https://$HOST_NAME \
+    --set rancherReplicas=$REPLICAS \
+    --set rancherBootstrapPassword=$BOOTSTRAP_PASSWORD \
+    --set rancherIngressClassName=nginx \
+    --set global.aws.accountNumber=$AWS_ACCOUNT_ID \
+    --set global.aws.roleName=$ROLE_NAME
+  ```
+
+  :::note
+
+  Monitor the rancher-cloud pod logs as the rancher-cloud pod is deleted 1 minute after a successful or failed installation.
+
+  :::
+
+  ```shell
+  kubectl logs -f $POD -n cattle-rancher-csp-deployer-system
```

Comment:
Is this example command intended to be for the preceding note? If so, then it should be grouped with the note and the actual pod name should be inserted instead of $POD.